packages/design-system/src/styles/components/_Button.scss

```diff
@@ -147,26 +143,20 @@
 
   &:focus,
   &.ds-c-button--focus {
-    @if $ds-include-focus-styles {
-      @include focus-styles;
-    } @else {
-      border-color: rgba($border-color-inverse, 0.8);
-      color: rgba($color-base-inverse, 0.8);
-    }
+    @include focus-styles;
+    background-color: $color-background-inverse;
```

Comment:
Needed to set the background color for inverse buttons to use the same as the default state of inverse buttons